### DISCUSSION TITLE
Personnalisation de l'iconographie des objets

### DISCUSSION THREAD

**Discussion by ghost** [dbarzin/mercator#304] [Ideas]
Bonjour,

Tout d'abord merci pour l'application et le temps consacré à son développement.

Vu dans la roadmap quelques fonctionnalités qui vont nous intéresser (champs personnalisés, nature des flux, etc.) pour formaliser nos échanges de données inter applicatifs.

J'ai lu aussi qu'il était prévu une refonte des icônes.
Par extension, serait-il possible de personnaliser l'icône d'un objet afin de visuellement identifier plus facilement son usage ?

Pour reprendre, les exemples sur la nature d'un flux ("saisie manuelle", "transfert de fichier", "partage de fichier", "api", etc.), la personnalisation de l'iconographie nous permettrait d'identifier chaque objet et son usage plus facilement .

Merci d'avance.

**Comment by dbarzin** [comment]
Bonjour Vincent,

Merci, je suis content d'apprendre que cette application vous est utile.

Pour les icônes, le travail est fastidieux car il existe une trentaine d'objets dans la cartographie. Souhaitez-vous changer les icônes pour quelques objets uniquement ? Lesquels ?

Pour la nature des flux, cela va être disponible en février.

Cordialement,
Didier

**Reply by ghost** [reply]
Bonjour,

Merci pour ta réactivité.

La nature des flux devrait suffire pour notre usage mais je pensais aussi aux services applicatifs.

Cordialement,
Vincent

**Reply by dbarzin** [reply]
La nature des flux est en place.
Cordialement,
Didier

**Reply by ghost** [reply]
Merci.
